c/meterpreter/source/extensions/stdapi/server/sys/config/config.c

```diff
@@ -274,6 +274,41 @@ DWORD request_sys_config_drop_token(Remote* pRemote, Packet* pPacket)
 	return dwResult;
 }
 
+/*
+ * @brief Updates an existing thread token.
+ * @param pRemote Pointer to the \c Remote instance.
+ * @param pRequest Pointer to the \c Request packet.
+ * @returns Indication of success or failure.
+ */
+DWORD request_sys_config_update_token(Remote* pRemote, Packet* pPacket)
+{
+	Packet* pResponse = met_api->packet.create_response(pPacket);
+	DWORD dwResult = ERROR_SUCCESS;
+	HANDLE hToken = NULL;
+	HANDLE hDupToken = NULL;

```

Comment:
This isn't used anywhere and looks like it can be removed.

Reply:
Done https://github.com/rapid7/metasploit-payloads/pull/648/commits/bdd28855713ca6832de67c79edac0772671c97c7